app/invocation/invocation_action_card.tsx

```diff
@@ -0,0 +1,64 @@
+import pako from "pako";
+import React from "react";
+import format from "../format/format";
+import InvocationModel from "./invocation_model";
+import rpcService from "../service/rpc_service";
+
+interface Props {
+  model: InvocationModel;
+  inProgress: boolean;
+  hash: string;
+  size: Long;
+}
+
+interface State {
+  loading: boolean;
+  contents: string;
+}
+
+export default class ActionCardComponent extends React.Component {
+  props: Props;
+
+  state: State = {
+    loading: false,
+    contents: ""
+  };
+
+  fetchAction() {
+    let actionFile = "bytestream://localhost:1987/blobs/" + this.props.hash + "/" + String(this.props.size);
+    rpcService
+      .fetchBytestreamFile(actionFile, this.props.model.getId())
+      .then((contents: any) => {
+        this.setState({
+            contents: contents,
+            loading: true,
+        })
+      })
+      .catch(() => {
+        console.error("Error loading bytestream timing profile!");
+        this.setState({
+          ...this.state,
+          timingLoaded: false,
```

Comment:
once you change the class definition to `React.Component<Props, State>` above, TypeScript should catch the error here :)